### ISSUE TITLE
Mask RCNN and RLE

### ISSUE THREAD

**Issue by edoardonemni** [waspinator/pycococreator#22]
Hello everyone,

I am running the matterport Mask_RCNN on my own dataset created using waspinator/pycococreator 

Everything works fine when I am using polygons, but as soon as I am using RLE because I have crowded images I got an error when trying to display a few images from the training dataset,

Is there a way to convert RLE to Polygons?

**Comment by waspinator** [comment]
you can decode RLE into binary masks using the [decode](https://github.com/cocodataset/cocoapi/blob/master/PythonAPI/pycocotools/mask.py#L32) function in the original cocoapi tools, and then use [binary_mask_to_polygon]9https://github.com/waspinator/pycococreator/blob/master/pycococreatortools/pycococreatortools.py#L35) to get a polygon. 

But the current coco format doesn't encode holes in objects properly, so this may not help solve your issue.

**Comment by gethubwy** [comment]
> Hello everyone,
> 
> I am running the matterport Mask_RCNN on my own dataset created using waspinator/pycococreator
> 
> Everything works fine when I am using polygons, but as soon as I am using RLE because I have crowded images I got an error when trying to display a few images from the training dataset,
> 
> I got problems with images with holes using polygons and I assumed that is why the pycococreator uses RLE when the image is crowded. This part works perfectly, but then since MaskRCCN requires polygons I am stuck in this loop.
> 
> Is there a way to convert RLE to Polygons? I am not even sure this would solve the problem

Hello,Excuse me, I want to know how to train a new mode, I use the software named "labelme" to label my data, I want to train a new model starting from pre-trained COCO weights, but I don't know how to write the annotation file in validate set, are there tools to help me? Thank you.

**Comment by enemni** [comment]
Hello @gethubwy, I am not sure I get your question, can you please share the code with us. labelme should solve your problem and label your entire dataset
